roles/proftpd/defaults/main.yml

```diff
@@ -15,3 +15,5 @@ manala_proftpd_configs:           []
 manala_proftpd_users_file:        /etc/ftpd.passwd
 manala_proftpd_users_template:    ~
 manala_proftpd_users:             []
+manala_proftpd_users_default_uid: 108
```

Comment:
Could you convert it to
```
manala_proftpd_users_defaults:
    uid: 108
    gid: 100
```
to keen consistance across roles :)

Reply:
Done ! thx